spec/moneyTrackerSpec.js

```diff
@@ -0,0 +1,44 @@
+describe("Moneytracker", function() {
+
+  var moneytracker;
+
+  beforeEach(function() {
+    moneytracker = new Moneytracker();
+  });
+
+  describe("#recordEarning", function() {
+    it("records (an) earning(s) which can be collated and printed as a running balance", function(){
+      moneytracker.recordEarning(100, "11/08/17");
+      expect(moneytracker.printBalance()).toEqual(console.log("Balance: 100"));
```

Comment:
What does `console.log` return? Is it necessary to call `console.log`? Try following the flow and getting visibility to find out.